pokemon-react-app/src/components/PokemonDetail.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router'
+import axios from 'axios'
+import { useDispatch, useSelector } from 'react-redux'
+import { notFoundPokemon, setPokemon } from '../redux/actions/pokemonActions'
+import { Img } from './../styles/Img.style'
+import { Container } from '../styles/Container.style'
+import { Type } from '../styles/Type.style'
+import { Label } from '../styles/Label.style'
+import { Card } from '../styles/Card.style'
+import NotFoundComponent from './NotFoundComponent'
+import { Container2, List, Item, LabelItem } from '../styles/StatsList.style'
+
+const PokemonDetail = () => {
+
+    const [isLoading, setLoading] = useState(true)
+    const { pokemonName } = useParams()
+    const dispatch = useDispatch()
+    const { pokemon, notFound } = useSelector(state => state.pokemon)
+
+    const fetchPokemonData = async () => {
+        const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`)
```

Comment:
Better use await approach with try/catch instead of mixing with promise/then/catch.